src/tools/webrtc-test-client/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Is this third-party code?  If so, can it be a submodule?

If this is being contributed directly to the Matter SDK, shouldn't it use the standard Matter SDK license?